spec/models/comment_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  describe 'Validating Comment Model' do
+    before(:each) do
+      @user = User.create(name: 'John', bio: 'Doctor', photo: 'No link', posts_counter: 2)
+      @post = Post.create(author_id: @user.id, title: 'Title', text: 'First post', likes_counter: 0,
+                          comments_counter: 0)
+      @comment = Comment.new(text: 'comment1', user_id: @user.id, post_id: @post.id)
+    end
+
+    it 'Should update comments counter' do
+      @comment.save
+      @post.reload
+      expect(@post.comments_counter).to eq(1)
+    end
+  end
+end
```

Comment:
- [ ] Good job 👏 . However,  all your Model specs are meant to  have  Validation tests as specified in the project requirements.

### Sample code:
```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```
### Instruction:
![image](https://github.com/sameedmohsin28/Blog-App/assets/50990925/02efdccf-6f1c-4199-a4f8-35af051427a9)

Reply:
Could you kindly elaborate further on what you mean by that? I can not understand that.

Reply:
Hello @sameedmohsin28 , i clearly gave a sample code to show you what's needed. You need to add testing for all validation made to the models. 
- The one I added  showed you a spec testing for the name validation which you added in the user model.

Reply:
I have already added the tests for all the validations in my opinion which includes the example you gave. That's why I asked if I was missing out on something. Below is the screenshot which contains name validation for User in highlighted text. Please tell me what else do I need to add more.

![image](https://github.com/sameedmohsin28/Blog-App/assets/122236545/25786e58-fd47-488c-a989-23f9867a1990)

Reply:
Please use the `describe` rspec grouping for the validations from the requirements. Check the sample I gave. Add same for all other files, likes, post and users.

Reply:
Oh ok. I get it. Thanks!